### PR TITLE
Add rename OmSupplyApi -> GraphQLClient

### LINKED ISSUE #976: Remove sdk/api from useOmSupplyApi
With the new implementation of colocated sdks within packages, the global `useOmSupplyApi` hook doesn't really make sense.

Should be changed to be something more like `useGraphQLClient` / `GraphQLClientProvider` - without the sdk/api there's no need to make it omSupply specific

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #976 

- Renamed `OmSupplyApi` -> `GraphQLClient`

Realised there were other options 😭  Think I prefer `GqlClient` or `Gql` 😁 
`useGqlClient` / `useGql` / `GqlProvider` / `GqlClientProvider`

What about you?

### CODE REVIEW

File: packages/common/src/api/OmSupplyApiContext.tsx [hunk 1]
```diff
@@ -2,36 +2,36 @@ import React, { FC, useMemo, useEffect, useState, useCallback } from 'react';
 import { createContext } from 'react';
 import { GraphQLClient } from 'graphql-request';
 
-export const createOmSupplyApi = (url: string): { client: GraphQLClient } => {
+export const createGraphQLClient = (url: string): { client: GraphQLClient } => {
```
Comment (mark-prins) [inline packages/common/src/api/OmSupplyApiContext.tsx:5]: This file could be renamed too 😃

**Review by mark-prins — APPROVED** [review]
👍  the name is fine, though `gql` along with `api` is pretty neat I agree

**Comment by joshxg** [comment]
So you like `useGql` / `GqlProvider` /  .. `const {client} = useGql()`  ? 😁

**Comment by mark-prins** [comment]
> So you like `useGql` / `GqlProvider` / .. `const {client} = useGql()` ? 😁

Yeah I do. ( GraphQLClient is fine too! doesn't need to change )

**Pull request merged**